### PR TITLE
chore: add history TTL property panel entry

### PR DESCRIPTION
**Pull request by marstamm**
related to https://github.com/camunda/camunda-modeler/issues/3712

Recording 2023-07-17 at 11 36 03

### CODE REVIEW

**Comment by philippfromme** [comment]
Now I remember why we did prevent selecting the root element. It's not supposed to be selected. No selected element leads to the root element being displayed in the properties panel.

brave_ys0j7sw1P8

**Comment by marstamm** [comment]
🤯

Let's select none then 😄

**Comment by marstamm** [comment]
adjusted via [a53aa92](https://github.com/camunda/linting/pull/66/commits/a53aa927ba7e63e16e4493239023dfd05ba79233)

**Review by philippfromme — CHANGES_REQUESTED** [review]
👀

File: lib/modeler/Linting.js [hunk 1]
```diff
@@ -30,6 +30,8 @@ export default class Linting {
 
     if (element !== this._canvas.getRootElement()) {
       this._selection.select(element);
+    } else {
+      this._selection.select(null);
```
[the remark this answers is not in the document]
Comment (philippfromme) [reply lib/modeler/Linting.js:34]: Cf. https://github.com/bpmn-io/diagram-js/blob/develop/test/spec/features/selection/SelectionSpec.js#L127

[the remark this answers is not in the document]
Comment (marstamm) [reply lib/modeler/Linting.js:34]: Is it? In the selection module, we also use `null`: 
https://github.com/bpmn-io/diagram-js/blob/master/lib/features/selection/Selection.js#L36

[the remark this answers is not in the document]
Comment (marstamm) [reply lib/modeler/Linting.js]: Adjusted and removed the explicit `null` with [43e7520](https://github.com/camunda/linting/pull/66/commits/43e7520d8298d60103575e6b4d7737b67251c71c)

**Comment by nikku** [comment]
@philippfromme This is waiting for your review.

File: lib/modeler/Linting.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (philippfromme) [reply lib/modeler/Linting.js]: Okay, `select(null)` will actually result in the selection being `[]` as well but it's superfluous.

**Pull request merged**